samples/csharp/MicrosoftGraphSDK/Module.cs

```diff
@@ -0,0 +1,149 @@
+            }
+        }
+
+        /// <summary>
+        /// Publishes a Module, changes the state of an educationModule from its original draft status to the published status
+        /// </summary>
+        /// <param name="client">Microsoft Graph service client</param>
+        /// <param name="classId">User class id</param>
+        /// <param name="moduleId">User module id</param>
+        /// <returns>EducationModule</returns>
+        public static async Task<EducationModule> PublishAsync(
+            GraphServiceClient client,
+            string classId,
+            string moduleId)
+        {
+            try
+            {
+                return await client.Education
+                    .Classes[classId]
+                    .Modules[moduleId]
+                    .Publish
+                    .PostAsync();
+            }
+            catch (Exception ex)
+            {
+                throw new GraphException($"PublishAsync call: {ex.Message}", ex, classId, moduleId);
+            }
+        }
+
+        /// <summary>
+        /// Gets the Resource Collection Response of Module
+        /// </summary>
+        /// <param name="client">Microsoft Graph service client</param>
+        /// <param name="classId">User class id</param>
+        /// <param name="moduleId">User module id</param>
+        /// <returns>EducationModuleResourceCollectionResponse</returns>
+        public static async Task<EducationModuleResourceCollectionResponse> GetModuleResourcesAsync(
+            GraphServiceClient client,
+            string classId,
+            string moduleId)
+        {
+            try
+            {
+                return await client.Education
+                    .Classes[classId]
+                    .Modules[moduleId]
+                    .Resources.GetAsync();
+            }
+            catch (Exception ex)
+            {
+                throw new GraphException($"GetModuleResourcesAsync call: {ex.Message}", ex, classId, moduleId);
+            }
+        }
+
+        /// <summary>
+        /// Sets up the Resources Folder of Module
+        /// </summary>
+        /// <param name="client">Microsoft Graph service client</param>
+        /// <param name="classId">User class id</param>
+        /// <param name="moduleId">User module id</param>
+        /// <returns>EducationModule</returns>
+        public static async Task<EducationModule> SetupResourcesFolder(
+            GraphServiceClient client,
+            string classId,
+            string moduleId)
+        {
+            try
+            {
+                return await client.Education
+                    .Classes[classId]
+                    .Modules[moduleId]
+                    .SetUpResourcesFolder
+                    .PostAsync();
+            }
+            catch (Exception ex)
+            {
+                throw new GraphException($"SetupResourcesFolder call: {ex.Message}", ex, classId, moduleId);
+            }
+        }
+
+    }
+}
```

Comment:
Should we add examples for GetModules/Module, GetModuleResources/ModuleResource, PatchModule/ModuleResource, DeleteModule/ModuleResource

Reply:
Added the missing items Patch and Delete, Thanks for Suggestions